plugins/services/src/js/components/modals/NewCreateServiceModalForm.js

```diff
@@ -392,20 +358,27 @@ class NewCreateServiceModalForm extends Component {
   }
 
   getSectionContent(data, errorMap) {
-    const errorsAlertComponent = this.getErrorsAlertComponent();
+    const {showAllErrors} = this.props;
+    const errors = this.getErrors();
 
     if (this.state.isPod) {
       return [
         <TabView id="networking" key="multinetworking">
-          {errorsAlertComponent}
+          <ErrorsAlert
```

Comment:
Yes, I do see places where we could use `Tabs` without it being in a form. I like the ability to compose. It is easy to understand what is going on and is in line with how we are going about creating components now.